system/Cache/CacheInterface.php

```diff
@@ -58,6 +58,17 @@ public function delete(string $key);
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Deletes items from the cache store matching a given pattern.
+	 *
+	 * @param string $pattern Cache items glob like pattern
+	 *
+	 * @return mixed
+	 */
+	public function deleteMatching(string $pattern);
+
+	//--------------------------------------------------------------------
+
```

Comment:
You cannot add this to the interface as it would be breaking for others using this interface. Perhaps adding a default implementation of this method in the base class, instead?

Reply:
I understand the concern though I think it should be in the interface defined with the other cache methods. Having it anywhere else would be kind of a hack.
As for others using it, and correct me if I'm wrong, if this feature is merged, it would be at first in the develop branch that is meant for this kind of "unstable" change. On the next release, this would have to be communicated as potential breaking change.

Anyways, that's just my two cents, and I am fine with changing as requested!